docs/v2-system-addon/data_dictionary.md

```diff
@@ -135,6 +135,40 @@ Schema definitions/validations that can be used for tests can be found in `syste
   "click|block|pocket": 0
 }
 ```
+# Example Activity Stream `undesired_event` Logs
+
+```js
+{
+  "topic": "activity-stream",
+  "client_id": "8095a611-4ee4-b94d-b7b6-37923a0faa69",
+  "shield_id": "",
+  "release_channel": "default",
+  "addon_version": "0.0.0",
+  "locale": "en-US",
+  "user_prefs": 3,
+  "session_id": "{ca150256-a6ca-4e43-a375-28ce5fa3bffc}",
+  "page": "about:newtab",
+  "value": 5,
+  "event": "topsites_missing_data",
+  "action": "activity_stream_undesired_event"
+}
+```
+```js
+{
+  "topic": "activity-stream",
+  "client_id": "8095a611-4ee4-b94d-b7b6-37923a0faa69",
+  "shield_id": "",
+  "release_channel": "default",
+  "addon_version": "0.0.0",
+  "locale": "en-US",
+  "user_prefs": 3,
+  "session_id": "{ca150256-a6ca-4e43-a375-28ce5fa3bffc}",
+  "page": "about:newtab",
+  "value": 10,
+  "event": "highlights_missing_data",
+  "action": "activity_stream_undesired_event"
+}
+```
```

Comment:
I'm thinking there's a bunch of value in, when possible, having the event names describe what the data is exactly without having to refer to the documentation.  For example, I think highlights_missing_data probably wants to be called something like "highlights_data_late_by_ms" or something similar.